variables.tf

```diff
@@ -483,6 +483,15 @@ variable "bitbucket_elasticsearch_replicas" {
   type        = number
   default     = 2
 }
+variable "bitbucket_shared_home_snapshot_id" {
+  description = "EBS Snapshot ID with shared home content."
+  type        = string
+  default     = null
+  validation {
+    condition     = var.bitbucket_shared_home_snapshot_id == null || can(regex("^.{1,255}$", var.bitbucket_shared_home_snapshot_id))
```

Comment:
```suggestion
    condition     = var.bitbucket_shared_home_snapshot_id == null || can(regex("^snap-\\w{17}$", var.bitbucket_shared_home_snapshot_id))
```

Reply:
Should Every snapshot start with `snap-`?

Reply:
Yes, see https://github.com/atlassian-labs/data-center-terraform/pull/202#discussion_r852535213